util/pkg/env/standard.go

```diff
@@ -58,6 +58,8 @@ func BuildSystemComponentEnvVars(spec *kops.ClusterSpec) EnvVars {
 	vars.addEnvVariableIfExist("OS_DOMAIN_ID")
 	vars.addEnvVariableIfExist("OS_USERNAME")
 	vars.addEnvVariableIfExist("OS_PASSWORD")
+	vars.addEnvVariableIfExist("OS_APPLICATION_CREDENTIAL_ID")
+	vars.addEnvVariableIfExist("OS_APPLICATION_CREDENTIAL_SECRET")
```

Comment:
Maybe move at the end of the list, same as in `nodeup/pkg/model/protokube.go` ?

Reply:
Duplicate now?

Reply:
Noticed that. Already pushed an amendment. thanks